test/e2e/view-caseload.js

```diff
@@ -64,21 +64,21 @@ describe('View your caseload flow', () => {
     it('with the correct tabs and tables', () => {
       return browser.url(teamDefaultUrl + '/caseload')
         .waitForExist('[href="#overall"]')
-        .waitForExist('.sln-table-caseload')
+        .waitForExist('#overall-enhanced')
         .waitForExist('[href="' + workloadOwnerDefaultUrl)
         .click('[href="#custody"]')
-        .waitForExist('.sln-table-caseload')
+        .waitForExist('#custody-enhanced')
         .click('[href="#community"]')
-        .waitForExist('.sln-table-caseload')
+        .waitForExist('#community-enhanced')
         .click('[href="#license"]')
-        .waitForExist('.sln-table-caseload')
+        .waitForExist('#license-enhanced')
     })
   })
 
   describe('should navigate to the LDU caseload screen', () => {
     it('with the correct table, breadcrumbs and export button', () => {
       return browser.url(lduDefaultUrl + '/caseload')
-        .waitForExist('.sln-table-caseload')
+        .waitForExist('.sln-table-caseload-by-grade')
```

Comment:
Should this be checking for multiple tables now?

Reply:
The next test tests all the tables and tabs, I've combined both into a single test